### DISCUSSION TITLE
Docs site redesign feedback!

### DISCUSSION THREAD

**Discussion by hairyhenderson** [hairyhenderson/gomplate#1730] [Show and tell]
👋 I'm [working on a redesign of the docs site](https://github.com/hairyhenderson/gomplate/pull/1729), and looking for feedback!

Here's the old site: https://docs.gomplate.ca

Here's the new site: https://deploy-preview-1729--gomplate-docs.netlify.app/

There is one functional differences that I'm interested in feedback on: in the old site, headers within a specific doc page would show up in the navigation side-bar. In the new design, they don't, but instead are visible through a modal table of contents (with more detail):
### old
<img width="295" alt="image" src="https://user-images.githubusercontent.com/2037086/236896045-1b464374-74a2-47d5-a3e1-959c1ff1de2b.png">

### new
<img width="678" alt="image" src="https://user-images.githubusercontent.com/2037086/236896323-760f67e5-dcd7-4f2b-be96-9c2d4d1ba43e.png">

What do people think about this? Does anybody care?

**Comment by hairyhenderson** [comment]
Given I haven't gotten much feedback (except ❤️ 😂), I've merged the PR. New site will be up within the next few minutes.

**Reply by Kalimero66** [reply]
I like the TOC but it requires a lot of clicking. If one wants to see the TOC, a click on the TOC button is required each time a new page is loaded.

is it possible to have the TOC always visible while browsing a page ? 

The crypto functions page has the TOC hiding the entire page because of a few functions with long signatures. Could this be formatted not to

**Reply by hairyhenderson** [reply]
@Kalimero66 thanks for the feedback.

I did notice a `What's on this page` section on the [hugo docs](https://gohugo.io/templates/section-templates/) which I think would solve the ToC problem, but the new theme I chose doesn't seem to support this. I'll see if I can get some help from the authors...

Those `SHA*` function titles are pretty brutal, I agree. I'll see about reworking those.

**Reply by hairyhenderson** [reply]
I've filed https://github.com/hairyhenderson/gomplate/issues/1734 for the latter issue.

**Reply by Kalimero66** [reply]
Thank you for the update ... I've been using the new site and I do like: 
- the TOC 
- the CopyToClipboard 
- the Dark theme
